CoinControl/Models/AccountType.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  AccountType.swift
+//  CoinControl
+//
+//  Created by Md. Asadul Islam on 15/2/24.
+//
+
+import Foundation
+//MARK: sshs
+enum AccountType:String,CaseIterable {
+    case cash
+    case card//dummy comment
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>warning: (spaceInsideComments) Add leading and/or trailing space inside comments.